src/main/java/org/jabref/gui/preftabs/PreviewPrefsTab.java

```diff
@@ -176,8 +177,9 @@ public void setValues() {
         chosenModel.clear();
         boolean isPreviewChosen = false;
         for (String style : previewPreferences.getPreviewCycle()) {
-            if (CitationStyle.isCitationStyleFile(style)) {
-                chosenModel.addElement(CitationStyle.createCitationStyleFromFile(style));
+            Optional<CitationStyle> citationStyle = CitationStyle.createCitationStyleFromFile(style);
```

Comment:
Can you add a comment, why this works?

The `CitationStyle.isCitationStyleFile(style)` is completely removed. Is this contained in ` CitationStyle.createCitationStyleFromFile(style)`?

Do you have a clue, why `CitationStyle.isCitationStyleFile(style)` doesn

Reply:
I'll add a comment.

You are right, `CitationStyle.isCitationStyleFile(style)` is checked in `CitationStyle.createCitationStyleFromFile(style)`. It works as expected and in case the given style is not a valid file, an empty `Optional` is returned. 

Before the returned `Optional` was added to the `JList` without unwrapping it, causing the error in the referenced issue.